src/App.module.css

```diff
@@ -7,13 +7,50 @@
 }
 
 .image {
-    width: 400px;
+    width: 300px;
 }
 
 .progressLine {
     width: 300px;
 }
 
+.mainField {
+    position: relative;
+}
+
+.modalWindow {
+    border: solid black 1px;
+    border-radius: 5px;
+    background-color: white;
+    z-index: 10;
+
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    width: 250px;
+    height: auto;
+    text-align: center;
+    padding: 15px;
+}
+
+.wrapperCloseButton {
+position: relative;
+}
```

Comment:
pls try to adhere to consistent formatting (indents)

Reply:
fixed